lib/testing-helpcenter/data/page_title_list.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'json'
+# Open JSON file and parse it's data to PageTitleList object
+class PageTitleList
+  attr_reader :hash
+
+  def initialize(file_path)
```

Comment:
You can use `lib/testing-helpcenter/data/page_titles.json` file as default argument for `file_path` var